Mathlib/Topology/PartitionOfUnity.lean

```diff
@@ -575,4 +622,60 @@ theorem exists_isSubordinate [NormalSpace X] [ParacompactSpace X] (hs : IsClosed
   let ⟨f, hf⟩ := BumpCovering.exists_isSubordinate hs U ho hU
   ⟨f.toPartitionOfUnity, hf.toPartitionOfUnity⟩
 
+/-- If `X` is a locally compact T2 topological space and `U` is a locally finite open covering of a
+compact set `s`, then there exists a `PartitionOfUnity ι X s` that is subordinate to `U`. -/
+theorem exists_isSubordinate_of_locallyFinite_t2space [LocallyCompactSpace X] [T2Space X]
+    (hs : IsCompact s) (U : ι → Set X) (ho : ∀ i, IsOpen (U i)) (hf : LocallyFinite U)
+    (hU : s ⊆ ⋃ i, U i) : ∃ f : PartitionOfUnity ι X s, f.IsSubordinate U ∧
+    ∀ i, HasCompactSupport (f i) :=
```

Comment:
For readability, could you avoid starting the return type on the same line as an hypothesis if you are going to break a line in the middle?
```suggestion
    (hU : s ⊆ ⋃ i, U i) :
    ∃ f : PartitionOfUnity ι X s, f.IsSubordinate U ∧ ∀ i, HasCompactSupport (f i) :=
```
Same everywhere else

Reply:
I see, done